pymc_marketing/clv/models/beta_geo.py

```diff
@@ -344,3 +344,61 @@ def expected_num_purchases_new_customer(
         return (left_term * right_term).transpose(
             "chain", "draw", "t", missing_dims="ignore"
         )
+
+    def probability_of_n_purchases_up_to_time(
+        self,
+        t: Union[np.ndarray, pd.Series],
+        n: int,
+    ):
+        r"""
+        Compute the probability of n purchases.
+         .. math::  P( N(t) = n | \text{model} )
```

Comment:
```suggestion
         .. math::  P( N(t) = n \mid r, \alpha, a, b)
```
Perhaps just to stay aligned with the docstrings of the other methods